src/ConfigProvider.php

```diff
@@ -15,11 +15,29 @@ final class ConfigProvider
     public function __invoke(): array
     {
         return [
-            'dependencies'       => $this->getDependencies(),
+            'dependencies' => $this->getDependencies(),
+            /**
+             * The top-level configuration key for defining custom view helpers.
+             *
+             * This option should use the `ServiceManagerConfiguration` array format
+             */
+            'view_helpers'       => [],
             'view_manager'       => [
+                /**
+                 * Encoding passed to the Escaper and possibly used in other view-related configuration
+                 */
                 'encoding' => 'utf-8',
+
+                /**
+                 * The base path is provided to the BasePath view helper. This is the historic location for this
+                 * configuration item defined in MVC apps.
+                 */
+                'base_path' => null,
```

Comment:
laminas-mvc cannot use the new major version of laminas-view, so the question arises if we need the configuration key `view_manager` at all.
Related to https://github.com/laminas/technical-steering-committee/pull/223

Reply:
I've moved some stuff to `view_helper_config` anyway, and retained searching in `view_manager` for fewer BC issues.

#281 refactors template resolvers, so we'll be writing factories soon to set those up. MVC puts all of this under `view_manager`, Mezzio under `templates` (Both at the top-level).

I'm starting to wonder if we could do with a top-level key that is specific to this library such as `laminas-view` where we nest everything else…